### PR TITLE
Updated a recurring event & changed the what is in you text

### PR DESCRIPTION
**Pull request by satyapriyamishra222**
Signed-off-by: satyapriyamishra222 <satyapriyamishra222@gmail.com>
- Added a recurring event (Weekly community sync up meeting)
- Changed the What's in for you text

### CODE REVIEW

File: website/public/locale/en/translation.yaml [hunk 1]
```diff
@@ -188,7 +188,7 @@ home:
     optoro: optoro
     orange: orange
   whatsInItForYou:
-    title: What’s in for you
+    title: Why should you use OpenEBS ?
```
Comment (Pallavi-PH) [inline website/public/locale/en/translation.yaml:191]:
```suggestion
    title: Why should you use OpenEBS?
```

Comment (satyapriyamishra222) [reply website/public/locale/en/translation.yaml]: Done

**Review by Pallavi-PH — APPROVED** [review]

File: website/src/components/EventSlider/index.tsx [hunk 2]
```diff
@@ -66,10 +66,10 @@ const EventSlider: React.FC<EventsProps> = ({
             const dateB: any = new Date(eventA?.date);
             return sortOrder === "asc" ? dateB - dateA : dateB + dateA;
           })
-        : eventsData;
-      const allEvents = [...eventsData, ...recurringEvents];
-      setFilteredEvents([...allEvents]);    
+        : eventsData;   
     }
+    const allEvents = [...eventsData, ...recurringEvents];
+    setFilteredEvents([...allEvents]);
```
Comment (rakeshPRaghu) [inline website/src/components/EventSlider/index.tsx:72]:
```suggestion
    setFilteredEvents(allEvents); 
```

Comment (IsAmrish) [reply website/src/components/EventSlider/index.tsx:72]: since we are already using rest operator on one line above :)

**Review by IsAmrish — APPROVED** [review]

File: website/src/components/EventSlider/index.tsx (hunk 2 above)
Comment (satyapriyamishra222) [reply website/src/components/EventSlider/index.tsx:72]: That throws a type incompatible error.

Comment (satyapriyamishra222) [reply website/src/components/EventSlider/index.tsx:72]: Array are reference types. So we should we spread here.

**Pull request merged**